### PR TITLE
Fix/#31 success handling

### LINKED ISSUE #17: Add description to welcome screen
If the user opens the Assistify Outlook plugin for the first time, the user just sees an input for the server URL. At this point we need some explanation about the plugin, like "Welcome to the Assistify Outlook Plugin. With this plugin you are able to shift emails to Assistify in order to discuss emails with your colleagues. At first, please add your server information."

### LINKED ISSUE #31: Success handling (notification/navigation)
Once a discussion has been successfully created, a success message (e. g. a toast) shall be displayed

Ideally, a navigation to the target channel should be executed

### PR DESCRIPTION
**Pull request by vickyokrm**
closes #17,  #31 and also #32. Shows the success page after the mail posted.

### CODE REVIEW

File: assets/success.svg [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 50 50" style="enable-background:new 0 0 50 50;" xml:space="preserve">
+<circle style="fill:#25AE88;" cx="25" cy="25" r="25"/>
+<polyline style="fill:none;stroke:#FFFFFF;stroke-width:2;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;" points="
+	38,15 22,33 12,25 "/>
+<g>
```
Comment (jschirrmacher) [inline assets/success.svg]: What is the use of this lot of groups?

File: src/commands/commands.js [hunk 2]
```diff
@@ -3,21 +3,20 @@
  * See LICENSE in the project root for license information.
  */
 
-var config;
 var configEvent;
+var config;
```
Comment (jschirrmacher) [inline src/commands/commands.js]: Why did you change the order of the variables?

**Review by jschirrmacher — CHANGES_REQUESTED** [review]

File: src/commands/commands.js (hunk 2 above)
Comment (vickyokrm) [reply src/commands/commands.js]: ;)

File: assets/success.svg (hunk 1 above)
Comment (vickyokrm) [reply assets/success.svg]: No Idea. Just used the downloaded svg as is.

**Review by jschirrmacher — APPROVED** [review]
